modules/aioseop_sitemap.php

```diff
@@ -1370,13 +1421,52 @@ function get_sitemap_data( $sitemap_type, $page = 0 ) {
 					$sitemap_data = $this->get_term_priority_data( get_terms( $sitemap_type, $this->get_tax_args( $page ) ) );
 				} else {
 					if ( is_array( $this->extra_sitemaps ) && in_array( $sitemap_type, $this->extra_sitemaps ) ) {
-						$sitemap_data = apply_filters( $this->prefix . 'custom_' . $sitemap_type, $sitemap_data, $page, $this_options );
+						/**
+						 * Filter to update the sitemap data corresponding to the custom sitemaps that were added before.
+						 *
+						 * @since ?
+						 *
+						 * @param array $sitemap_data {
```

Comment:
The param `sitemap_data` had more data in it.

```
/**
 * Filter to update the sitemap data.
 *
 * @since ?
 *
 * @param array $sitemap_data {
 *      An array of arrays, where each array corresponds to the <url> element of the sitemap.
 *
 *      @type array $index {
 *         The sitemap data corresponding to an <url> element. Each key should correspond to the name of
 *         the element as specified by the sitemap XSD.
 *
 *         @type string $changefreq Amount of time to update.
 *         @type string $loc        URL location of the Sitemap.
 *         @type string $priority   Set between "0.0" and "1.0".
 *     }
 * }
 * @param string $sitemap_type   Name of the sitemap.
 * @param int       $page           Page number.
 * @param array $options {
 *      Array of the module's options.
 *
 *      TODO This needs a reference, and reference also needs to be documented. Options is ~98 variables/keys.
 * }
 */
```